app/src/main/java/com/infomaniak/swisstransfer/di/ApplicationModule.kt

```diff
@@ -18,11 +18,26 @@
 
 package com.infomaniak.swisstransfer.di
 
+import com.infomaniak.multiplatform_swisstransfer.SwissTransferInjection
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.CoroutineScope
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
 object ApplicationModule {
 
-//    @Provides
-//    @Singleton
-//    fun providesSwissTransferInjection() {
-//        // TODO: Implement this method
-//    }
+    @Provides
+    @Singleton
+    fun providesSwissTransferInjection() = SwissTransferInjection()
```

Comment:
Why not just `SwissTransfer` instead of `SwissTransferInjection` ?

Reply:
Because the name SwissTransfer might be confusing, given that the application is called SwissTransfer.
On the other hand, I've just seen the iOS solution, so let's talk about it again tomorrow when you're in the office.